maquetado/css/styles.css

```diff
@@ -34,3 +33,35 @@
     display: none;
   }
 }
+
+.features {
+  display: flex;
+  margin-top: 100px;
```

Comment:
you don't need this margin, the padding in the previous section should be enough